testing/test_capture.py

```diff
@@ -831,7 +833,7 @@ def test_dontreadfrominput():
 
 
 @pytest.fixture
-def tmpfile(testdir):
+def tmpfile(testdir) -> Generator[BinaryIO, None, None]:
```

Comment:
Generally better to use `Iterator[X]` over `Generator[X, None, None]` - shorter and stricter.

Hmm but now that I think about it, `pytest.fixture` actually would want a real generator, I think? If so, then yeah, better keep it.